src/libpgagroal/prometheus.c

```diff
@@ -942,6 +991,7 @@ home_page(int client_fd)
    data = append(data, "            <li>initial</li>\n");
    data = append(data, "            <li>max</li>\n");
    data = append(data, "            <li>active</li>\n");
+   data = append(data, "            <li>awaiting (on hold due to <i>blocking_timeout</i>)</li>\n");
```

Comment:
But it should be tied to *limits*, so why make a different metric? We have the total and per limit one. The only thing is that due to b131180 and 0767d9e it is now not tied to `blocking_timeout` anymore, so the description should be fixed (and also the documentation).
Otherwise please explain better what it is required to be done.

Reply:
I see the metrics that is backed by `blocking_timeout` as metrics that you likely are going to have alerts on - say you deploy applications that tries to way more connections that they are actually assigned.

Ideally `pgagroal_limit_awaiting` should all show up as `0` - `1` or `2` would likely be ok, but more and you need to investigate why you are getting all these requests.

Having this part of the existing metric will mean that you will have to dig out the information from the attribute.

Sure, both solutions are close to each other - but having it separate highlights that you may want to create a different graph/alert on it

Reply:
Moved in a separate piece of code via 0a4ab18, documentation added.